experiment/cherrypicker/README.md

```diff
@@ -0,0 +1 @@
+This project has moved to [https://github.com/kubernetes/test-infra/tree/master/prow/external-plugins/cherrypicker](https://github.com/kubernetes/test-infra/tree/master/prow/external-plugins/cherrypicker)
```

Comment:
@kargakis Is this okay for the breadcrumb?

Reply:
yeah, thanks